mcstatus/tests/test_querier.py

```diff
@@ -38,6 +38,28 @@ def test_query(self):
         }
         assert response.players.names == ["Dinnerbone", "Djinnibone", "Steve"]
 
+    def test_query_handles_unicode_motd_with_nulls(self):
+        self.querier.connection.receive(
+            bytearray(
+                b"\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00hostname\x00\x00*K\xd5\x00gametype\x00SMP\x00game_id\x00MINECRAFT\x00version\x001.16.5\x00plugins\x00Paper on 1.16.5-R0.1-SNAPSHOT\x00map\x00world\x00numplayers\x000\x00maxplayers\x0020\x00hostport\x0025565\x00hostip\x00127.0.1.1\x00\x00\x01player_\x00\x00\x00"
+            )
+        )
+        response = self.querier.read_query()
+        conn_bytes = self.querier.connection.flush()
+        
+        assert response.raw['game_id'] == 'MINECRAFT'
```

Comment:
Let's also verify that the motd is being read successfully:
```suggestion
        assert response.raw['game_id'] == 'MINECRAFT'
        assert response.motd == "\x00*KÕ"
```
Is this the expected motd?